internal/stack/logs.go

```diff
@@ -48,7 +58,7 @@ func copyDockerInternalLogs(serviceName, outputPath string) error {
 
 	outputPath = filepath.Join(outputPath, serviceName+"-internal")
 	serviceContainer := p.ContainerName(serviceName)
-	err = docker.Copy(serviceContainer, "/usr/share/elastic-agent/state/data/logs/default", outputPath)
+	err = docker.Copy(serviceContainer, "/usr/share/elastic-agent/state/data/logs/", outputPath)
```

Comment:
Dump before and after 8.5.0 seem to use different paths, are we ok with that? Or should we remove "default" from the path when present for consistency between versions?

Reply:
Before 8.6.0 (where the Unified Agent log was introduced), logs were copied from the `default` folder. Since that version, that folder does not longer exist.

I thought updating the code to copy from the upper directory `logs` would be backwards compatible with versions <8.6.0.

```
 $ tree elastic-stack-dump-8.8.0-SNAPSHOT/logs/elastic-agent-internal/
elastic-stack-dump-8.8.0-SNAPSHOT/logs/elastic-agent-internal/
└── elastic-agent-20230505.ndjson

0 directories, 1 file

 $ tree elastic-stack-dump-8.5.0/logs/elastic-agent-internal/
elastic-stack-dump-8.5.0/logs/elastic-agent-internal/
├── default
│   ├── filebeat-20230505-1.ndjson
│   ├── filebeat-20230505-2.ndjson
│   ├── filebeat-20230505-3.ndjson
│   ├── filebeat-20230505.ndjson
│   ├── metricbeat-20230505-1.ndjson
│   └── metricbeat-20230505.ndjson
└── elastic-agent-20230505.ndjson

1 directory, 7 files
```

The difference for versions <8.6.0 is that it would copy one file more (in the example `/usr/share/elastic-agent/state/data/logs/elastic-agent-20230505.ndjson`)  instead of copying just the ones from the default folder.

Reply:
Thanks for the clarification, understood :+1:

I would keep it as it is (adding that default folder). As you mention, those files in `default` folder will not be part of the logs for > 8.6.0 versions.